src/Monad/Identity.php

```diff
@@ -1,17 +1,19 @@
 <?php
-namespace Monad;
+namespace Widmogrod\Monad;
 
-use Common;
-use FantasyLand;
+use Widmogrod\Common\PointedTrait;
+use Widmogrod\Common\ValueOfInterface;
+use Widmogrod\Common\ValueOfTrait;
+use Widmogrod\FantasyLand;
 
 class Identity implements
-    FantasyLand\Monad,
-    Common\ValueOfInterface
+    FantasyLand\Monad ,
+    ValueOfInterface
 {
-    const of = 'Monad\Identity::of';
+    const of = 'Widmogrod\Monad\Identity::of';
 
-    use Common\PointedTrait;
-    use Common\ValueOfTrait;
+    use PointedTrait;
+    use ValueOfTrait;
```

Comment:
Using `namespaces` help helps me in understanding wether interface belong to a good place or not (especially during development), and as a side effect, there is less imports. If I may ask you to revert this change, at least for now.